app/controllers/application_controller.rb

```diff
@@ -1,3 +1,9 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+
+
+  def hogehoge
+
+    llll = 'lfasldkf'
```

Comment:
__[staging][RuboCop]__

Lint/UselessAssignment: Useless assignment to variable - `llll`.


<!-- Comment by [Sider](https://sider.review) -->